src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -213,3 +212,35 @@ export default createPaginationContainer(
     `,
   }
 )
+
+interface RelayProps {
```

Comment:
This component pretty much needs what got moved into the stand-alone `relayInterfaces`.